pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/KitRequestShipping.java

```diff
@@ -1748,4 +1748,9 @@ public boolean isKitRequiringTrackingScan() {
     public boolean hasTrackingScan() {
         return StringUtils.isNotBlank(trackingId);
     }
+
+    // todo arz test me
+    public boolean hasBSPCollaboratorParticipantId() {
+        return StringUtils.isNotBlank(bspCollaboratorParticipantId);
```

Comment:
I saw the check for blank bsp collaborator participant id in a few places so I extracted a method for it.